notebooks/7_reshape_table_layout.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>I don't quite follow what the last sentence in this paragraph means. I would recommend either cutting that sentence or perhaps demonstrating it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I'll cut this as it is not adding added value explaining all these options in detail.</p>